lib/improver/wind_direction.py

```diff
@@ -333,34 +335,51 @@ def wind_dir_decider(self, where_low_r, first_member):
                 Array of boolean values. True where original wind direction
                 estimate has low confidence. These points are replaced
                 according to self.backup_method
-            first_member (np.array):
-                Array of wind direction data from the first ensemble
+            first_member (iris.cube.Cube):
```

Comment:
Is this really the first realization any more?  It doesn't look like that's how this function is called.

Reply:
Corrected. See response to Gavin's review.